lib/__tests__/styleDictionary.test.js

```diff
@@ -0,0 +1,487 @@
+    });
+
+    it('should map text assets', () => {
+      const mapping = mapToStyleDictionaryTokens({ textStyles: inputTextStyles });
+
+      expect(mapping).toBeDefined();
+      expect(mapping).toHaveProperty('asset.font', outputTextStyles.asset.font);
+    });
+
+    it('should map text colors', () => {
+      const mapping = mapToStyleDictionaryTokens({ textStyles: inputTextStyles });
+
+      expect(mapping).toBeDefined();
+      expect(mapping).toHaveProperty( 'color.font', outputTextStyles.color.font);
+    });
+  });
+
+  describe('-> #mapBorders', () => {
+    const inputBorderStyles = [
+      {
+        id: 'inside',
+        token: 'border',
+        color: reusableColors[0].input,
+        width: 1,
+        type: 'INSIDE',
+        radius: 3
+      },
+      {
+        id: 'outside',
+        token: 'border',
+        color: reusableColors[1].input,
+        width: 2,
+        type: 'OUTSIDE',
+        radius: 12.5
+      },
+    ];
+    
+    const outputBorderStyles = {
+      color: {
+        border: {
+          inside: reusableColors[0].output,
+          outside: reusableColors[1].output,
+        }
+      },
+      size: {
+        border: {
+          inside: { value: '1' },
+          outside: { value: '2' },
+        },
+      },
+    };
+
+    it('should map border sizes', () => {
+      const mapping = mapToStyleDictionaryTokens({ borders: inputBorderStyles });
+
+      expect(mapping).toBeDefined();
+      expect(mapping).toHaveProperty('size.border', outputBorderStyles.size.border);
+    });
+
+    it('should map border colors', () => {
+      const mapping = mapToStyleDictionaryTokens({ borders: inputBorderStyles });
+
+      expect(mapping).toBeDefined();
+      expect(mapping).toHaveProperty('color.border', outputBorderStyles.color.border);
+    });
+  });
+
+  describe('-> #mapShadows', () => {
+    const inputShadowStyles = [
+      {
+        id: 'simple',
+        token: 'shadow',
+        blur: 3,
+        x: 0,
+        y: 1,
+        spread: 0,
+        type: 'inner-shadow',
+        color: reusableColors[0].input,
+      },
+      {
+        id: 'simple',
+        token: 'shadow',
+        blur: 4,
+        x: 0,
+        y: 2,
+        spread: 5,
+        type: 'drop-shadow',
+        color: reusableColors[1].input,
+      },
+    ];
+
+    const outputShadowStyles = {
+      color: {
+        shadow: {
+          simple: {
+            innerShadow: reusableColors[0].output,
+            dropShadow: reusableColors[1].output,
+          }
+        }
+      }
+    }
+
+    it('should map shadow colors', () => {
+      const mapping = mapToStyleDictionaryTokens({ shadows: inputShadowStyles });
+
+      expect(mapping).toBeDefined();
+      expect(mapping).toHaveProperty('color.shadow', outputShadowStyles.color.shadow);
+    });
+  });
+
+  describe('sample output file mapping', () => {
+    it('should correctly map the __mocks__/sample_output.json file', () => {
```

Comment:
Please remove/define this empty test case

Reply:
Ah, forgot to remove. Was planning on creating a test for the sample output, but seemed like a lot of overhead each time the mock sketch file changes for little gain. Will remove.